src/main/scala/Scratchpad.scala

```diff
@@ -20,7 +20,7 @@ trait CanHaveBackingScratchpad { this: BaseSubsystem =>
   private val portName = "Backing-Scratchpad"
 
   val spadOpt = p(BackingScratchpadKey).map { param =>
-    val spad = LazyModule(new TLRAM(address=AddressSet(param.base, param.mask), beatBytes=mbus.beatBytes))
+    val spad = mbus { LazyModule(new TLRAM(address=AddressSet(param.base, param.mask), beatBytes=mbus.beatBytes, devName=Some("scratchpad"))) }
```

Comment:
Can we call it `backingScratchpad` or something? It should be clear that this is a very special scratchpad not a generic RAM.